app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   has_many :videos, through: :user_videos
 
   validates :email, uniqueness: true, presence: true
-  validates_presence_of :password
+  validates_presence_of :password_digest
   validates_presence_of :first_name
   enum role: %i[default admin]
   has_secure_password
```

Comment:
Not sure if both `validates_presence_of :password_digest` and `has_secure_password` are required. Thoughts?